CityOfDetroit/bloom
# fix: listing list dupes in pagination

## Issue

- Closes #1078

## Description

This PR fixes the issue where we were seeing duplicate listings and other listings missing in the partner portal. I assumed the issue was caused by how TypeORM was handling the joins with pagination, but I believe I was able to narrow it down to ordering, which gets applied to both the inner filter query and the outer query and the differences caused by the what was the application due/open date order by clause. I thought it made more sense for the list to default ordering to name, so you'll see that update as well as some other unrelated updates like removing the need for joining leasingAgentListings and instead filtering on listing IDs from the user's profile.

## How Can This Be Tested/Reviewed?

Carefully, set your local up to point to production, start up the app and paginate through listings in the partner portal. You shouldn't see duplicates anymore.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
